try {
    processarDados();
} catch (erro) {
    console.log(erro);
}